static/js/app.js

```diff
@@ -276,10 +276,54 @@ $(function() {
             return params;
         },
         setMultipleMarkersIcon: function() {
+            var markerGroups = [];
+            var markerObj;
+            var groupID = 1;
+            var groupsColours = [];
             _.each(this.oms.markersNearAnyOtherMarker(), function(marker) {
                 marker.icon = getIcon("multiple", SEVERITY_VARIOUS);
```

Comment:
Yes, this one. It's being overwritten later anyway - in what is now the `switch` statement, so in what sense is it a fallback?